Kickstarter-iOS/Views/Controllers/RewardsCollectionViewController.swift

```diff
@@ -232,6 +232,16 @@ extension RewardsCollectionViewController {
   ) {
     if let rewardCell = cell as? RewardCell {
       rewardCell.delegate = self
+      rewardCell.addCollectionViewController(to: self)
```

Comment:
Why do you think it's best to configure addition of the VC in `RewardsCollectionViewController`? I think that it's the case that the pills only show in specific cases, so it might make more sense to have the `RewardViewModel` drive the creation & addition of the `PillCollectionViewController` via an output. Something like this:

**In RewardCell**
```
self.viewModel.outputs.addPillCollectionViewControllerWithData
  .observeForUI()
  .observeValues { data in
    self.addCollectionViewController()
    self.reloadCollectionViewControllerWith(data)
}
```

Reply:
Yeah that would work too...the problem is that for addition we need to parent VC in order for the containment to work properly...we'd have to store a `weak` reference to it in the cell.

What do you think about that?